docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: "2.1"
```

Comment:
what good is this setup to the devs, who want to be able to run the app in docker based on the current code they have open on their machines? why would they want to run whatever old version is on docker hub at aypas/*?

Reply:
You're right, I'm gonna work on making this more convenient.